.github/workflows/release_dependency_versions.yml

```
@@ -7,20 +7,17 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-permissions:  # added using https://github.com/step-security/secure-workflows
-  contents: read
-
 jobs:
   trigger_chromedriver:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # tag: v3
     - name: Trigger New chromedriver Release
       run: |
-        if [[ ${{ github.event.release.tag_name }} =~ ^v[0-9]+\.0\.0$ ]]; then
+        if [[ ${{ github.event.release.tag_name }} =~ ^v(\d+\.)(\d+\.)(\d+)$ ]]; then
```

no need for the captures, i think?
```suggestion
        if [[ ${{ github.event.release.tag_name }} =~ ^v\d+\.\d+\.\d+$ ]]; then
```